### ISSUE TITLE
Fails to start with Home Assistant 2022.3.0b5 (unexpected keyword argument 'new_state')

### ISSUE THREAD

**Issue by Steltek** [IATkachenko/HA-SleepAsAndroid#39]
### Home Assistant
**Version:** 2022.3.0b5
**Installation method** (according to [instructions](https:/hass/www.home-assistant.io/installation/)):
[X] Operating System
[ ] Container
[ ] Supervised
[ ] Core
### Integration

**Installation method:**
[X] HACS
[ ] manual from release
[ ] manual from git
**Integration version:** v1.8.2
**Root topic:** 
### Application settings

**topic:**
[X] I can see MQTT messages from application at MQTT-server
## How it works now?

The integration used to work. (MQTT broker settings etc all fine and have not changed.)

Upon startup, the integration errors out with "TypeError: async_subscribe_topics() got an unexpected keyword argument 'new_state'"

## Debug log
```
2022-03-01 14:18:50 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up sleep_as_android platform for sensor
Traceback (most recent call last):
  File "/hass/homeassistant/lib/python3.9/site-packages/homeassistant/helpers/entity_platform.py", line 249, in _async_setup_platform
    await asyncio.shield(task)
  File "/hass/.homeassistant/custom_components/sleep_as_android/sensor.py", line 48, in async_setup_entry
    await instance.subscribe_root_topic(async_add_entities)
  File "/hass/.homeassistant/custom_components/sleep_as_android/__init__.py", line 257, in subscribe_root_topic
    self._subscription_state = await subscribe_2021_07(
  File "/hass/.homeassistant/custom_components/sleep_as_android/__init__.py", line 235, in subscribe_2021_07
    return await subscription.async_subscribe_topics(
TypeError: async_subscribe_topics() got an unexpected keyword argument 'new_state'

```

**Comment by IATkachenko** [comment]
Thank you for report.
Will check it. Looks like version comparison goes bad :(

**Comment by IATkachenko** [comment]
Nice
```python
f"{ha_version.version} >= {AwesomeVersion('2022.3.0')} is {ha_version.version >= AwesomeVersion('2022.3.0')}"
```
```
 2022.3.0b5 >= 2022.3.0 is False
```

I should read documentation carefully...

**Comment by IATkachenko** [comment]
Only 2022.3.0 beta installations affected. Will fix soon.

**Comment by IATkachenko** [comment]
Fixed in [v1.8.3](https://github.com/IATkachenko/HA-SleepAsAndroid/releases/tag/v1.8.3).
Because only beta version was affected this release will be marked as pre-release and will be available only if "get beta version" option is turned on for the integration in HACS.
